src/main/java/ua/goit/gojava32/kickstarter/service/SendMail.java

```diff
@@ -0,0 +1,48 @@
+package ua.goit.gojava32.kickstarter.service;
+
+import java.util.Properties;
+import javax.mail.Message;
+import javax.mail.MessagingException;
+import javax.mail.PasswordAuthentication;
+import javax.mail.Session;
+import javax.mail.Transport;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+
+public class SendMail {
+
+  private static final String username = "kickstartergoit32@ukr.net";
```

Comment:
passwords and connection details should be not hardcoded, they will differ in development / local / production environments usually